src/commons/utils/JsSlangHelper.ts

```diff
@@ -76,10 +76,11 @@ function cadetAlert(value: any) {
  *
  * @param list the list to be visualised.
  */
-function visualiseList(list: any) {
+function visualiseList(...xs: any[]) {
```

Comment:
I think prof @martin-henz suggested during one of our meetings that we should standardize and spell visualize with a Z instead of S.